packages/blocks/chart/README.md

```diff
@@ -0,0 +1,94 @@
+# Block template
```

Comment:
Let's remove all the template contents. We can just add something like

```
# Chart block
```

A couple of sentences with upstream links would be also useful.